blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/datastax/DatastaxShardStateIO.java

```diff
@@ -54,7 +54,7 @@
             Select.Where statement = QueryBuilder
                     .select()
                     .all()
-                    .from(CassandraModel.QUOTED_KEYSPACE, CassandraModel.CF_METRICS_STATE_NAME)
+                    .from(CassandraModel.CF_METRICS_STATE_NAME)
```

Comment:
Where does it refer the keyspace from? I mean how does it know to look for this column family in keyspace "DATA"?